client/src/actions/index.js

```diff
@@ -49,3 +49,10 @@ export const login = data => {
       });
   };
 };
+
+export const logout = () => {
```

Comment:
Note that this action does not require input or deliver a payload - it simply wipes out the JWT from localStorage and makes a request to the reducer to clear out the auth state.